src/neo/SmartContract/Native/OracleContract.cs

```diff
@@ -96,7 +96,7 @@ public long GetPrice(DataCache snapshot)
         }
 
         [ContractMethod(RequiredCallFlags = CallFlags.States | CallFlags.AllowCall | CallFlags.AllowNotify)]
-        private void Finish(ApplicationEngine engine)
+        private async void Finish(ApplicationEngine engine)
```

Comment:
Maybe we need to change https://github.com/neo-project/neo/blob/e51c8776f7b6b83d99c01055c55c38651af4461c/src/neo/SmartContract/Native/NativeContract.cs#L119 to handle `async` methods

Reply:
We don't need. These asynchronous methods do not affect the return value.